pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -52,16 +52,20 @@ in mkDerivation rec {
   dontWrapGApps = true;
   dontWrapQtApps = true;
 
-  nativeBuildInputs = [ pkg-config cmake ninja python3 wrapGAppsHook wrapQtAppsHook removeReferencesTo ];
+  nativeBuildInputs = [
+    pkg-config cmake ninja python3 wrapGAppsHook wrapQtAppsHook removeReferencesTo
```

Comment:
removeReferencesTo doesn't seem to be used

Reply:
Removed.